.github/workflows/python-cleanliness.yml

```diff
@@ -16,10 +16,15 @@ jobs:
         with:
           submodules: 'recursive'
 
+      # Run flake8 but exclude third-party libraries
+      - run: pipx run flake8 --exclude=MAVProxy/modules/mavproxy_cesium,MAVProxy/modules/lib
```

Comment:
We use `--show-source` in `run_flake8` - any reason not to include it here?

Note that cesium is external, but most, if not all, of `MAVProxy/modules/lib` is ours to clean.  Why did you choose to exclude it entirely here rather than add the `# flake8: noqa`?

Reply:
Yes, let's add `--show-source`.  (Ruff has no such option, so I have gotten out of the practice of adding it.)

https://github.com/ArduPilot/MAVProxy/tree/master/MAVProxy/modules says:
> mavproxy_cesium @ 78f7be3

Therefore, the code in `MAVProxy/modules/mavproxy_cesium` is a git submodule and is not in this repository.